fiscalsim_us/variables/gov/states/va/tax/income/va_adj_gross_income.py

```diff
@@ -24,13 +24,18 @@ def formula(tax_unit, period, parameters):
 
         subtotal = line3 - line8
 
-        if filing_status == 1 or filing_status == 3:
+        if (
+            filing_status == 0
+            or filing_status == 2
+            or filing_status == 4
+            or filing_status == 2
```

Comment:
@austinperryfrancis. This line is redundant. Remove it.

Reply:
I just made the change, instead of a second filing status 2, it's supposed to be filing status 3